allensdk/brain_observatory/ecephys/stimulus_table/naming_utilities.py

```diff
@@ -195,4 +195,15 @@ def map_column_names(table, name_map=None, ignore_case=True):
     output = table.rename(columns=name_map)
 
     return output
+
+
+def eval_str(val):
+    if isinstance(val, str):
+        try:
+            val = eval(val)
```

Comment:
I'm not sure using a try/except here is the best especially with the use of a bare except. Is there a way we can do this by checking for `isinstance` or `type`?

Reply:
Looks better. Thanks perhaps add a inline comment for what the first test is doing.